frontend/src/views/Pipelines/PipelineCreateForm.js

```diff
@@ -119,38 +119,37 @@ const PipelineCrateForm = (props) => {
               SamlGroupName: values.SamlGroupName,
               tags: values.tags,
               devStrategy: values.devStrategy
-
-            }
-          })
-        );
-        if (!response.errors) {
-          setStatus({ success: true });
-          setTriggerEnvSubmit(true);
-          setPipelineUri(response.data.createDataPipeline.DataPipelineUri);
-          setSubmitting(false);
-          enqueueSnackbar('Pipeline creation started', {
-            anchorOrigin: {
-              horizontal: 'right',
-              vertical: 'top'
-            },
-            variant: 'success'
-          });
-          navigate(
-            `/console/pipelines/${response.data.createDataPipeline.DataPipelineUri}`
+              }
+            })
           );
-        } else {
+          if (!response.errors) {
+            setStatus({ success: true });
+            setTriggerEnvSubmit(true);
+            setPipelineUri(response.data.createDataPipeline.DataPipelineUri);
+            setSubmitting(false);
+            enqueueSnackbar('Pipeline creation started', {
+              anchorOrigin: {
+                horizontal: 'right',
+                vertical: 'top'
+              },
+              variant: 'success'
+            });
+            navigate(
+              `/console/pipelines/${response.data.createDataPipeline.DataPipelineUri}`
+            );
+          } else {
+            setTriggerEnvSubmit(false);
+            dispatch({ type: SET_ERROR, error: response.errors[0].message });
+          }
+        } catch (err) {
+          console.error(err);
```

Comment:
Seems a leftover after debugging

Reply:
I am not going to resolve it as it might conflict with @dbalintx work on pipelines